starknet_py/net/account/account_client.py

```diff
@@ -450,6 +450,14 @@ async def create_account(
             supported_tx_version=version,
         )
 
+    async def get_nonce(
+        self,
+        contract_address: int,
+        block_hash: Optional[Union[Hash, Tag]] = None,
+        block_number: Optional[Union[int, Tag]] = None,
+    ) -> int:
+        return self._get_nonce()
```

Comment:
`get_nonce` checks nonce of an address provided as the argument. Internal _get_nonce should call either this method (in case of tx version = 1 ) or use the logic it is using now (tx version = 0).